src/pages/view/ViewDashboard.js

```diff
@@ -1,11 +1,13 @@
-import { useDhis2ConnectionStatus, useDataEngine } from '@dhis2/app-runtime'
+import {
```

Comment:
It's difficult to understand what behavior has changed in this component. Could you add an explanation to the PR description?

Reply:
I've added some text to the PR description now. Will resolve this comment.